src/UI/Settings/Notifications/Edit/NotificationEditView.js

```diff
@@ -11,14 +11,17 @@ var view = Marionette.ItemView.extend({
     template : 'Settings/Notifications/Edit/NotificationEditViewTemplate',
 
     ui : {
-        onDownloadToggle : '.x-on-download',
-        onUpgradeSection : '.x-on-upgrade',
-        tags             : '.x-tags'
+        onDownloadToggle             : '.x-on-download',
+        onUpgradeSection             : '.x-on-upgrade',
+        tags                         : '.x-tags',
+        indicator                    : '.x-indicator',
+        authorizedNotificationButton : '.AuthorizeNotification'
```

Comment:
Use `x` classes for events/UI referencing, following the same style, so `.x-authorize-notification`